### ISSUE TITLE
Incorrect timezone information in pdf metadata.

### ISSUE THREAD

**Issue by JohnLErvin** [PyFPDF/fpdf2#451]
Describe the bug

**Error details**
In resulting pdf metadata, the localtime is reported, but the UTC timezone is indicated. Either report local timezone are change to UTC time when saving in metadata.   

**Minimal code**
Please include some minimal Python code reproducing your issue:
```python
from fpdf import FPDF

pdf = FPDF()
pdf.add_page()
pdf.set_font('helvetica', size=12)
pdf.cell(txt="hello world")
pdf.output("hello_world.pdf")
```
Open the pdf in any pdf viewer, look at properties and notice timezone for created is incorrect. 

**Environment**
* Arch
* Python 3.10.2
* `fpdf2` 2.5.4

-->

**Comment by Lucas-C** [comment]
Thank you for reporting this @JohnLErvin!

I submitted #452 to fix this

**Comment by JohnLErvin** [comment]
You are welcome @Lucas-C, and thank you for the rapid change. I'll have a look to verify fix...

**Comment by Lucas-C** [comment]
Great.

There are some PDF files generated with `fpdf2` with the fix applied:
* https://github.com/PyFPDF/fpdf2/blob/master/test/metadata/setting_old_date.pdf
* https://github.com/PyFPDF/fpdf2/blob/master/test/alias_nb_pages.pdf

**Comment by Lucas-C** [comment]
This fix has been included in [release 2.5.5](https://github.com/PyFPDF/fpdf2/releases/tag/2.5.5)
